### PR TITLE
Documentation Added

### LINKED ISSUE #2: Project is lacking a README file.
This project is intended to be a "learning by doing" experience for the developer team. But to be more developer-friendly, we need a README file that showcases the basic usage of our app and the features of the DASH app. It should include the following details:

1.  Clicking on a bar in the bar chart makes changes in the adjacent pie chart. (APP Feature)
2.  Clicking on a legend item of the pie chart makes that item disappear (You can use this feature to sell total vs partially vaccinated). (DASH Feature)
3. In the state comparison tab, selecting no state will result in the line chart of India. (APP Feature)
4. In the state comparison tab, the Search option is available. (DASH Feature)

### PR DESCRIPTION
**Pull request by Adheena19**
Readme is updated.

### CODE REVIEW

**Comment by VaishU2235** [comment]
I have checked your work and found the following issues.
1) The refresh rate of gif is so fast. The user is unable to see each frame.
2) As we have decided to limit the responsiveness of the plots for a better experience for the user, please remove comments related to zoom-in/out from the documentation.
3) Please try to be more clear and concise. For eg: 
          Clicking on a bar in the bar chart makes changes in the adjacent pie chart.
This statement can be amended as follows:
          Clicking on a particular state's bar from the bar chart changes the adjacent pie chart and the vaccination status of that particular state will be shown there.
4) Add a small bio of each developer if possible.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,82 @@
...
+
+
+  
+## Demo
+ 
+![ezgif com-gif-maker (2)](https://user-images.githubusercontent.com/86158829/136935766-a61f3ca4-ce1d-4fb0-8179-91f13134753f.gif)
+
+  
+
+## Documentation
```
Comment (VaishU2235) [inline README.md]: Guidelines is a better title

File: README.md [hunk 2]
```diff
@@ -1,3 +1,82 @@
...
+This project aims to create a covid webapp.It is a "learning by doing"experience which provided us the practical experience in handling of git,flask,plotly,dash,deployment etc.This project was done with the data from COVID INDIA API.This covid webapp provides the visualizations of vaccination status in the states of India along with the statewise Death rate,Tested Cases,Confirmed Cases,Recovered Cases,Deceased Cases.Thus making  an efficient way of comparison.
```
Comment (VaishU2235) [inline README.md]: This part is repeated

File: README.md [hunk 3]
```diff
@@ -1,3 +1,82 @@
...
+![covidwebapp](https://user-images.githubusercontent.com/86158829/136670586-05271c6a-27fd-40f1-adec-17026fc62bd8.png)
+
+![coviviz](https://user-images.githubusercontent.com/86158829/136839294-3d6c8912-ed34-4d00-b2a3-0afdb152fd43.png)
+
+## Deployment
+
+To see the deployed project, follow the link
+
+```bash
+  https://coviviz.herokuapp.com/.
```
Comment (VaishU2235) [inline README.md]: It is https://coviviz.herokuapp.com/ , not https://coviviz.herokuapp.com/. .
Please Take care of the "." symbol at the end

**Review by VaishU2235 — CHANGES_REQUESTED** [review]
Please see comments made in lines 23 24 and 59

File: README.md (hunk 3 above)
Comment (Adheena19) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (Adheena19) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Adheena19) [reply README.md]: Done

**Pull request merged**